src/client/Microsoft.Identity.Client.Brokers/RuntimeBroker.cs

```diff
@@ -13,6 +13,7 @@
 using Microsoft.Identity.Client.Instance.Discovery;
 using Microsoft.Identity.Client.Internal.Broker;
 using Microsoft.Identity.Client.Internal.Requests;
+using Microsoft.Identity.Client.NativeInterop;
```

Comment:
Did you E2E test this manually yet? I see that `WamAdapter.ParseRuntimeResponse` hardcodes "Bearer" token type.

Reply:
You can use `authResult.IsPopAuthorization` to determine if the auth result in Bearer or POP.